src/api/uploads/MultiputPart.js

```diff
@@ -298,10 +298,6 @@ class MultiputPart extends BaseMultiput {
      * @return {void}
      */
     cancel(): void {
-        if (this.xhr && typeof this.xhr.abort === 'function') {
```

Comment:
We don't need to abort since it's done in this.destroy below

Reply:
Can you add the same safety check around the abort in super.destroy()?

Reply:
According to @DanDeMicco flow should be protecting us here.